mkt/reviewers/templates/reviewers/includes/review_history.html

```diff
@@ -62,36 +62,33 @@
                   </td>
                 </tr>
               {% endif %}
-              {% if version.approvalnotes %}
-                <tr>
-                  <th>{{ _('Notes for Reviewers') }}</th>
-                  <td class="activity_approval">
-                    <div class="history-notes">
-                      {{ version.approvalnotes|urlize(100)|nl2br }}
-                    </div>
-                  </td>
-                </tr>
-              {% endif %}
+              {# No need to show approvalnotes, that'll be handled by commbadge, see below. #}
             </thead>
             <tbody>
               {# Results populated here by reviewers_commbadge.js (Commbadge API) #}
+              <tr class="loading-msg">
+                <td>{{ _('Loading App History &hellip;')|safe }}</td>
+              </tr>
+            </tbody>
+            <tfoot>
+              {# Paginator will be displayed if necessary. #}
               <tr class="comm-notes-paginator">
                 <th>&nbsp;</th>
                 <td>
-                  <a class="button next" href="#">{{ _('Next Page') }}</a>
-                  <a class="button prev" href="#">{{ _('Previous Page') }}</a>
+                  <a class="button next" href="#">{{ _('Next Comments') }}</a>
```

Comment:
Notes is more accurate as some of them won't be comments (automatic messages) but I don't think it matters too much.